mitosheet/src/components/endo/widthUtils.tsx

```diff
@@ -116,4 +116,21 @@ export const reconciliateWidthData = (prevWidthData: WidthData | undefined, oldC
     }
 
     return getWidthData(sheetData, oldWidths);
+}
+
+/*
+    Guess the minimum column width such that the entire column header and each cell is displayed
+    completely withour wrapping.
+
+    Returns a number of pixels 
+*/
+export const guessFullWidth = (sheetData: SheetData, columnIndex: number, displayColumnHeader: string): number => {
```

Comment:
`guessFullWidthOfColumnHeaderOrContent` is a more informative name I won't wonder about in future, I think

Reply:
E.g. much clearer, given we have functions like `getWidthData`